tmt-app/src/Styling/Home.css

```diff
@@ -0,0 +1,206 @@
+}
+
+.Home_Hero_Text{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+}
+.Home_Hero_Text_1{
+    width: 75%;
+    height: 100%;
+}
+.Home_Hero_Text_1 h1{
+    color: #1B1D2E;
+    font-family: 'Poppins';
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 38px;
+    margin:0;
+    padding-bottom: 1.5rem;
+    text-align: center;
+}
+.Home_Hero_Text_2{
+    width: 45%;
+    height: 100%;
+}
+.Home_Hero_Text_2 h6{
+    color: #1B1D2E;
+    font-family: 'Poppins';
+    font-style: normal;
+    font-weight: 600;
+    font-size: 14px;
+    line-height: 18px;
+    text-align: center;
+    margin: 2rem 0;
+}
+
+
+/*
+    Home page Buttons Stlying
+*/
+.Home_Button_Container{
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: row;
+}
+.Home_Button_Main{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    height: auto;
+}
+.Home_Button_Main_Btn{
+    font-family: 'Poppins';
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 48px;
+    color: #FFFFFF;
+    background-color: #73956F;
+    border-radius: 8px;
+    border: 0;
+    padding: 1rem 5rem;
+    cursor: pointer;
+}
+.Home_Button_Main_Btn:hover{
+    background-color: #732C2C;
+}
+
+.Home_Button_Main_Link{
+    font-family: 'Poppins';
+    font-style: normal;
+    font-weight: 600;
+    font-size: 18px;
+    line-height: 30px;
+    color: #1B1D2E;
+    text-decoration-line: underline;
+    cursor: pointer;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-top: 4rem;
+}
+
+.Home_Button_Main_Link svg{
+    padding-top: 3px;
+    padding-left: 10px;
+}
+
+/*
+    Background Styling for Button Section
+        -Bottom of home page
+*/
+.Home_Button_Bg_Container{
+    position: absolute;
+    bottom: -125%;
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    z-index: -1;
+}
+.Home_Button_Bg_Left{
+    position: relative;
+}
+.Home_Button_Bg_Right{
+    position: relative;
+    right: -2.5%;
+}
+#Home_Button_Bg_Img1{
+    width: 150%;
+    vertical-align: middle;
+    transform: rotate(133.23deg);
+}
+#Home_Button_Bg_Img2{
+    width: 250%;
+    transform: rotate(32.27deg);
+    position: absolute;
+    left: -60%;
+    top: 15%;
+}
+#Home_Button_Bg_Img3{
+    width: 100%;
+    transform: rotate(-33.7deg);
+}
+
+@media screen and (min-width: 750px){
+    .Home_Hero_Text_1 h1{
+        font-size: 48px;
+        line-height: 66px;
+    }
+    .Home_Hero_Text_2 h6{
+        font-size: 18px;
+        line-height: 26px;
+    }
+}
+
+@media screen and (min-width: 1350px){
+    .Home_Hero_Text_1 h1{
+        font-size: 64px;
+        line-height: 86px;
+    }
+    .Home_Hero_Text_2 h6{
+        font-size: 24px;
+        line-height: 36px;
+    }
+    
+}
+
+
```

Comment:
Same comment as Michael, check out the MUI library in a little more depth and understand what it has to offer, I think we could cut out 150 lines of css if we're using MUI more

Reply:
Nice, we cut out a lot from this after adding in some of the material components.

I think a good end goal to keep in mind is to write little to no css. working with something like MaterialUI , we can automate a ton of the styling and only write custom styling when there are some really custom demands that Material can't do for us